### ISSUE TITLE
Points of Interest

### ISSUE THREAD

**Issue by theo-armour** [fgx/fgx.github.io#15]
@geoffmcl 

This is a continuation of:

https://github.com/fgx/fgx.github.io/issues/11/ 'Encoded binary paths anyone?'

and

https://github.com/fgx/fgx.github.io/issues/4 / Use of issues, and tag, releases

Let's start with some possibilities:

All the following have _ad hoc_ titles. Please do suggest better titles. Ditto categories and their descriptions.
## 1. Traffic Control Waypoints

There's a bunch of waypoint data here (perhaps in need of an update, but nevermind for now):

https://github.com/fgx/fgx-airports/tree/gh-pages/csv-data

Whenever a new flightpath demo is being created, these above files can be searched and and data lying within a certain boundary could be extracted written to a small file just containing the local data.

The Flightpath script would open the extracted data file and display relevant data on 3D placards at the  designated lat, lon and elevation. The display of these ( and any other placards ) could be toggled on and off.
## 2. Other Local Waypoints

Names of peaks, rivers, large buildings and other large things visible from the air would also be good things to indicate on the flightpath viz.

If the waypoint data happens to contain polyline or other geographic data then this could be embossed on the bitmap overlay or used to create a 3D mesh. 

It may be possible to obtain some of this type of data from the Google Maps API - as well as from a number of GIS sources including FlightGear. These are all possibilities is to be explored.

Again the goal would be to create a data file - probably in roughly the same format as the waypoints files - with relevant and useful local data that can be quickly and easily loaded into the flightpath simulator.
## 3. Flight Instructions

These are the instructions a flight instructor might give. They could include preflight checklist items and instructions during the flight.

Very often such instructions are related to time and location, so it seems logical to embed these in the Flightpath CSV file

An alternative might be to put the instructions in a separate file but then the time stamps and or location of reach record would have to be coordinated

---

How is this as a starting point? Are the items well-defined? Are there other types or categories of geographic data that we should consider displaying?

**Comment by theo-armour** [comment]
@geoffmcl 

Below is some of your text copied from the binary-encoded issue

```

> VFR Reference points
> We still seem very different, very far apart, on this ;=()
> 
> You point to fgx/fgx-airports, but what we have there are all IFR things... navaids, runways, ils, fixes... nothing to do with this VFR circuit of LEIG...
> 
> VFR - Visual Flight Rules - fly by what a human can see out the window on the ground
> IFR - Instrument Flight Rules - Fly by instruments, with no window at all.
> Well IFR can still use VFR for take off and landing, but after that, like in the Concord, the window is closed and it is all by instruments... waypoint to waypoint...
> 
> So again out of your 1, 2, 3 types, only maybe 2 would apply, but even then very selective of Other Local Waypoints... only those visible objects to fly this particular flight... not a general tourist map of the area...
> 
> So still need some discussion, understanding and agreement on this...
> 
> PS: For a few days now, having internet connection problems, so can not post this now... will try later... its working now, so...

```

> VFR Reference points

It's good you mentioned VFR. Somehow I had no idea no idea that this is what we were talking about. ;-)

But this error points to what's actually happening: I have no idea what is needed and I have been groping in the dark trying to discover what it's all about. ;-(

It might be easier if you write a little specification about what you need or want. Or just send me some data and I'll figure it out. 

In other words, I should shut up and let you take the lead here...
